Sriram/Reg.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>Registration page</title>
+  </head>
+  <body>
+    <h2><strong>Register</strong></h2>
+
+    <form id="Form">
+      First name:
+      <input type="text" name="firstname" value="First name" required/>
+      Last name:
+      <input type="text" name="lastname" value="Last name" /><br /><br />
+      Gender:
+      <label for="male">Male</label>
+      <input type="radio" name="gender" value="male" /> Male
+      <label for="female">Female</label>
+      <input type="radio" name="gender" value="female" /> Female
+      <label for="other">Other</label>
+      <input type="radio" name="gender" value="other" /> Other<br /><br />
```

Comment:
Only the label is required. Additional text will end up in duplication.

Reply:
@Nuthi-Sriram This issue still persists.

Reply:
Added required

Reply:
The one that you are seeing says outdated

Reply:
@Nuthi-Sriram Only label is required. Not extra text.

Reply:
I didn't understand

Reply:
> I didn't understand

Open the page in your browser.

Reply:
Yeah I did
![image](https://user-images.githubusercontent.com/46497115/66508222-4018e480-eaee-11e9-935b-5da26fdbf445.png)

Reply:
> Yeah I did
> ![image](https://user-images.githubusercontent.com/46497115/66508222-4018e480-eaee-11e9-935b-5da26fdbf445.png)

Each gender comes twice. Only the label is required.

Reply:
Done